demo/Example+SLRealizer+Usage.ipynb

```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import absolute_import\n",
+    "from __future__ import division\n",
+    "from __future__ import print_function\n",
+    "\n",
+    "import os, sys\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "realizer_path = os.path.join(os.environ['SLREALIZERDIR'], 'slrealizer')\n",
+    "sys.path.insert(0, realizer_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to interface with SLRealizer\n",
```

Comment:
Great! Thanks for writing this tutorial, @jiwoncpark :-)

Reply:
:)